### ISSUE TITLE
Exception inside application: Incoming message has no 'type' attribute

### ISSUE THREAD

**Issue by Clashsoft** [oTree-org/oTree#72]
The following error seems to appear commonly in the room wait page.

OTree version is 2.5.8, running on Heroku.

```java
2020-06-19T10:55:21.885466+00:00 app[web.1]: 10:55:21 asgiserver | ERROR Exception inside application: Incoming message has no 'type' attribute
2020-06-19T10:55:21.888065+00:00 app[web.1]: 10:55:21 asgiserver |   File "/app/.heroku/python/lib/python3.7/site-packages/channels/sessions.py", line 183, in __call__
2020-06-19T10:55:21.888301+00:00 app[web.1]: 10:55:21 asgiserver |     return await self.inner(receive, self.send)
2020-06-19T10:55:21.888512+00:00 app[web.1]: 10:55:21 asgiserver |   File "/app/.heroku/python/lib/python3.7/site-packages/channels/middleware.py", line 41, in coroutine_call
2020-06-19T10:55:21.888780+00:00 app[web.1]: 10:55:21 asgiserver |     await inner_instance(receive, send)
2020-06-19T10:55:21.889035+00:00 app[web.1]: 10:55:21 asgiserver |   File "/app/.heroku/python/lib/python3.7/site-packages/channels/consumer.py", line 59, in __call__
2020-06-19T10:55:21.889488+00:00 app[web.1]: 10:55:21 asgiserver |     [receive, self.channel_receive], self.dispatch
2020-06-19T10:55:21.889982+00:00 app[web.1]: 10:55:21 asgiserver |   File "/app/.heroku/python/lib/python3.7/site-packages/channels/utils.py", line 52, in await_many_dispatch
2020-06-19T10:55:21.890308+00:00 app[web.1]: 10:55:21 asgiserver |     await dispatch(result)
2020-06-19T10:55:21.890745+00:00 app[web.1]: 10:55:21 asgiserver |   File "/app/.heroku/python/lib/python3.7/site-packages/channels/consumer.py", line 71, in dispatch
2020-06-19T10:55:21.891008+00:00 app[web.1]: 10:55:21 asgiserver |     handler = getattr(self, get_handler_name(message), None)
2020-06-19T10:55:21.891265+00:00 app[web.1]: 10:55:21 asgiserver |   File "/app/.heroku/python/lib/python3.7/site-packages/channels/consumer.py", line 19, in get_handler_name
2020-06-19T10:55:21.891534+00:00 app[web.1]: 10:55:21 asgiserver |     raise ValueError("Incoming message has no 'type' attribute")
2020-06-19T10:55:21.892441+00:00 app[web.1]: 10:55:21 asgiserver |   Incoming message has no 'type' attribute
```

**Comment by oTree-org** [comment]
Hello, thank you for reporting. I think I have found the issue and fixed it. I have published it in the latest beta release (2.6.0b16). For more info on the beta see here: https://otree.readthedocs.io/en/latest/misc/version_history.html#version-2-6-beta

**Comment by Clashsoft** [comment]
Where can I view this change? I didn't find a commit. Also, where (on GitHub) can I track releases? This repo has [none](https://github.com/oTree-org/oTree/releases) and the core repo also [hasn't been updated](https://github.com/oTree-org/oTree-core/releases) in a while.
